lib/translator/inquiry.h

```diff
@@ -48,5 +48,7 @@ void TranslateUnitSerialNumberVpd(
 
 void TranslateSupportedVpdPages(absl::Span<uint8_t> buffer);
 
+void TranslateBlockDeviceCharacteristicsVpd(absl::Span<uint8_t> buffer);
```

Comment:
Once #76 adds an anonymous namespace, putting this function there instead of the header is appropriate imo